src/localization.js

```diff
@@ -266,7 +294,6 @@ angular.module('ngLocalize')
                 updateHtmlTagLangAttr(lang);
 
                 $rootScope.$broadcast(localeEvents.localeChanges, currentLocale);
-                $rootScope.$broadcast(localeEvents.resourceUpdates);
```

Comment:
If resourceUpdates event will pass back the bundle objects, then this needs to be removed as bundles are not loaded here.